translations/stripes-smart-components/en.json

```diff
@@ -59,6 +59,7 @@
   "ll.permanent": "permanent",
   "ll.saveAndClose": "Save and close",
   "ll.cancel": "Cancel",
+  "ll.remoteLabel": "(Remote)",
```

Comment:
if you look at the `(${loc.code})` part, parenthesis leveraged to code there. Maybe it makes sense to be consistent with this suffix as well?